readthedocs/rtd_tests/tests/test_doc_building.py

```diff
@@ -266,7 +591,23 @@ def test_container_timeout(self):
             'Build exited due to time out')
         self.assertEqual(self.mocks.docker_client.exec_create.call_count, 1)
         self.assertTrue(build_env.failed)
+
+        # api() is not called anymore, we use api_v2 instead
         self.assertFalse(self.mocks.api()(DUMMY_BUILD_ID).put.called)
+        self.mocks.mocks['api_v2.build']().put.assert_called_with({
+            'id': DUMMY_BUILD_ID,
+            'version': self.version.pk,
+            'success': False,
+            'project': self.project.pk,
+            'setup_error': u'',
+            'exit_code': 1,
+            'length': 0,
+            'error': 'Build exited due to time out',
+            'setup': u'',
+            'output': u'',
+            'state': u'finished',
+            'builder': u'foo'
```

Comment:
All these listings of massive argument lists seem brittle (it will break whenever we add/change/remove/ an argument). Can we just check for the actual value we care about being called, presumably `error`?

Reply:
Test are failing because of this, in fact:

    'length': <ANY>

Besdies, it's complicated to find the differences when it fails. In case you want to leave as it is now, I'd suggest to use `assertDictEqual`